runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -31,8 +31,17 @@
 --- shouldn't be done directly in the change callback anyway as they will be very frequent. Rather
 --- a plugin that does any kind of analysis on a tree should use a timer to throttle too frequent
 --- updates.
+---
+
+-- Debugging:
+--
+-- vim.g.__ts_debug levels:
+--    - 1. Messages from languagetree.lua
+--    - 2. Parse messages from treesitter
+--    - 2. Lex messages from treesitter
+--
+-- Log file can be found in STDPATH('log')/treesitter.log
```

Comment:
why is STDPATH uppercase? :)
```suggestion
-- Log file can be found in stdpath('log')/treesitter.log
```